tasks/app/consistency.py

```diff
@@ -140,7 +140,7 @@ def cleanup_gitlab(context, dryrun=False, clean=False):
             if delta.total_seconds() <= GRACE_PERIOD:
                 skipped += 1
                 continue
-            success = current_app.git_backend.delete_remote_project(project)
+            success = current_app.git_backend._delete_remote_project(project)
```

Comment:
Calling a private method means that it's not really private. 
It would be clearer to make both methods public and have one of them called delete_remote_project_by_name

Reply:
I didn't change this until the cleanup gitlab job was failing... but I don't know whether it's still failing :thinking: Need to check.

Plus, I'm not sure what the difference between `delete_remote_project` and `_delete_remote_project` is :woman_facepalming: 

```
    def delete_remote_project(self, project_name):
        self._ensure_initialized()              
        project = self.get_project(project_name)
        if project:                             
            self._delete_remote_project(project)
                                              
    def _delete_remote_project(self, project):
        self._ensure_initialized()
        try:                                   
            self.gl.projects.delete(project.id)
            return True                 
        except gitlab.GitlabDeleteError:
            pass    
        return False
```

Why are they not combined...?

Reply:
good question, I think the first level answer is that one takes the project and one takes the guid (name) of the project.
Note I don't understand enough to know what this signifies. 
I guess you can't combine them as the above call calls the "inner" one, hence my suggestion of two interfaces.

Reply:
Oh yeah... I didn't realize that one takes a project_name, one takes a project :joy:  Thanks for pointing it out!